### PR TITLE
added logout button

### LINKED ISSUE #452: When logged in then also login option still displayed
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in our [Gitter channel](https://gitter.im/fossasia/susi_server)

**Current behavior:**
When the user has logged in then also `Login` is showing in the options
login chrome

**Expected behavior:**
When the user has logged in then `Login` should be replaced by `Log out`.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #452 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
logout

#### Changes proposed in this pull request:
Fixed logout toggle when the user login

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -622,17 +622,18 @@ window.onload = () => {
...
-        if (userDetails.loggedUser && userDetails.loggedUser.email) {
-            accessToken = userDetails.loggedUser.accessToken;
-            log.innerHTML = log.innerHTML.replace("Login", "Logout");
-            log.innerHTML = log.innerHTML.replace("login.svg", "logout.png");
-        } else {
-            log.innerHTML = log.innerHTML.replace("Logout", "Login");
-            log.innerHTML = log.innerHTML.replace("logout.png", "login.svg");
-        }
-    });
+    chrome.storage.sync.get("loggedUser",function(userDetails){
```
Comment (rossoskull) [inline src/script.js]: As arrow functions are used elsewhere, I think this function should be an arrow function for the sake of convention.

Comment (samagragupta) [reply src/script.js]: Made the changes

**Comment by samagragupta** [comment]
@gabru-md please review

**Comment by samagragupta** [comment]
@gabru-md @hkedia321 Please review

**Comment by fragm3** [comment]
This PR changes leads to more options in settings. 2 settings options are there in the drop down menu, fix them. Show settings only on logged in

Current:
screenshot 2019-01-23 at 3 43 34 am

This PR changes:
screenshot 2019-01-23 at 3 43 04 am
screenshot 2019-01-23 at 3 43 15 am

**Comment by samagragupta** [comment]
@fragm3 the commits are not squashing.
Can you not merge it directly?

**Comment by fragm3** [comment]
@samagragupta each PR for merge needs the approval of at least 2 mentors. Please rebase and merge commits from your end

**Comment by samagragupta** [comment]
@stealthanthrax @gabru-md  Please review.

**Comment by samagragupta** [comment]
@Orbiter @akshatnitd Please review.

**Comment by samagragupta** [comment]
@fragm3 Sorry, But I am not able to squash them, there is some problem.
Can you please merge it directly.

**Pull request merged**